plugin/src/utils.ts

```diff
@@ -5,3 +5,7 @@ export function isRelease() {
     process.env.NODE_ENV?.match(pattern)
   );
 }
+
+export function isWeb() {
+  return process.env.REANIMATED_BABEL_PLUGIN_IS_WEB === '1';
+}
```

Comment:
I don't like that being an environmental variable - I think it'd much better if it was passed as an option like `processNestedWorklets`.
cc @tomekzaw